src/getMaxLevelDepthFromMapping.ts

```diff
@@ -0,0 +1,79 @@
+import {
+  Cube,
+  DataVisualizationWidgetMapping,
+  LevelCoordinates,
+  getHierarchy,
+  getLevel,
+} from "@activeviam/activeui-sdk";
+import { quote } from "@activeviam/mdx";
+
+/**
+ * Returns the column width for the tree table calculated by the baseWidth + (maxLevelDepth * levelMultiplier)
+ */
+export function getTreeColumnWidth({
+  maxLevelDepth,
+  mapping,
+  treeTableColumnWidth,
+  columnLevels,
+  cube,
+}: {
+  mapping: DataVisualizationWidgetMapping;
+  maxLevelDepth: number;
+  treeTableColumnWidth: [number, number];
+  columnLevels: LevelCoordinates[];
+  cube: Cube;
+}): { [columnKey: string]: number } {
```

Comment:
Why do we call this `columnKey` instead of `columnWidth`?

Reply:
Looking at this more closely, I'm going to push another commit to rename this file. I renamed the function but not the file name 🤦‍♂️

Reply:
Okay thanks for the explanation